### ISSUE TITLE
Clip property is depreciated

### ISSUE THREAD

**Issue by Sora2455** [csstools/sanitize.css#183]
[The clip property is depreciated](https://developer.mozilla.org/en-US/docs/Web/CSS/clip).

The visually hidden code (`[aria-hidden="false"][hidden]:not(:focus)`) should have a `clip-path: inset(100%);` as well as the clip.

**Comment by jonathantneal** [comment]
This may be preserved for browser backward compatibility, or it may use this new syntax if transformed before publish. It’s good information to know, and my inability to add it would only be to maintain backward compatibility. Would you mind spinning up a CodePen to demonstrate the change?

**Comment by Sora2455** [comment]
Pen is here: https://codepen.io/Sora2455/pen/dEpxwz?editors=1100

(I found while writing that that `inset(100%)` doesn't seem to work, but `circle(0)` does.

**Comment by jonathantneal** [comment]
If we’re going to use something hacky, I’d like to keep using `rect(0, 0, 0, 0)` which already works without a fallback.
